src/zenml/utils/analytics_utils.py

```diff
@@ -333,11 +333,15 @@ def track(
         # infinite loop
         if gc._zen_store is not None:
             zen_store = gc.zen_store
+            user = zen_store.get_user()
```

Comment:
I haven't looked at this code in a while, so excuse the stupid question: This will now be (in most local cases) the ID of the default user of the local store, right? So deleting this store will give me a new user ID, but my client ID (which is also always tracked) will stay the same?

Reply:
Yes, that's exactly right !